packages/eslint-plugin/src/configs/react.js

```diff
@@ -216,6 +143,93 @@ const config = {
     'operator-assignment': 'off',
     'prefer-destructuring': 'off',
     'prefer-template': 'off',
+    // airbnb or other config overrides (some temporary)
+    // TODO: determine which rules we want to enable, and make needed changes (separate PR)
```

Comment:
Do we want to open a ticket for this right away?

Reply:
i dunno TBH. as stated in PR I didn't spent extra time to go through every rule rather colocating for better readability. Feel free to investigate further

Reply:
This comment is from back when we originally migrated to eslint. The idea back then was to extend the airbnb config (maybe because v0 did?) and eventually enable more rules as we thought it made sense, but we never got around to following up. Since we're disabling so many airbnb rules, it would probably make more sense to just directly enable the rules from there that we *do* want and remove that config extension.

Reply:
this ^ 100%. I'd like us to remove airbnb completely (unmaintaned, loooots of opinionated rules that are not useful that much. I'll work on this as a follow up.